ArduCopter/inertia.cpp

```diff
@@ -15,16 +15,16 @@ void Copter::read_inertia()
         return;
     }
 
-    // without home return alt above the EKF origin
-    if (!ahrs.home_is_set()) {
-        // with inertial nav we can update the altitude and climb rate at 50hz
-        current_loc.alt = inertial_nav.get_altitude();
+    Location::ALT_FRAME frame;
+    if (ahrs.home_is_set()) {
+        frame = Location::ALT_FRAME_ABOVE_HOME;
     } else {
-        // with inertial nav we can update the altitude and climb rate at 50hz
-        current_loc.alt = pv_alt_above_home(inertial_nav.get_altitude());
+        // without home return alt above the EKF origin
+        frame = Location::ALT_FRAME_ABOVE_ORIGIN;
     }
+    current_loc.set_alt_cm(inertial_nav.get_altitude(), frame);
+    current_loc.change_alt_frame(Location::ALT_FRAME_ABOVE_ORIGIN);
```

Comment:
Although I do agree with this, everyone is aware this is a real change from using altitude above home 99% of the time to using altitude above origin 100% of the time, right?

And this means that keeping the code above for using home altitude makes no sense since we'll always end up with altitude above origin.

Reply:
Changed this to home-relative.  Insane.

Reply:
I thought it was too easy, why did I say something? 😄

I wished the code to get the altitude used the AHRS but I know I won't convince you 😞😛